packages/pages/src/upgrade/upgrade.ts

```diff
@@ -0,0 +1,20 @@
+import { Command } from "commander";
+import path from "path";
+import { updatePages } from "./pages_updater.js";
+import { migrateConfigs } from "./migrate_config.js";
+import { execSync } from "child_process";
+
+const handler = async () => {
+  const rootPath = path.resolve("");
+  await updatePages(rootPath);
+  await migrateConfigs(rootPath);
+  execSync("pages generate templates");
+  execSync("pages generate artifacts");
+};
```

Comment:
Please test what happens after you run your script and then try to `npm run prod`. I'm pretty sure there might be errors due to have config.yaml and the older sites-config files (features.json). If there is an error, tell Spruce. Whether there's an error or not, we should clean up (delete) the old files/folders.

Reply:
npm run dev results in this error when opening a page:
`Error: Multiple documents found for locale: "en"`
<img width="1028" alt="image" src="https://github.com/yext/pages/assets/115099870/edf95497-a6b7-443b-8c48-953d29fbdcb8">

npm run prod results in this:
```
pages-starter-react-locations/node_modules/@yext/pages/dist/vite-plugin/build/closeBundle/closeBundle.js:53
Error: TypeError [ERR_INVALID_ARG_TYPE]: The "paths[1]" argument must be of type string. Received undefined
code: 'PLUGIN_ERROR',
  plugin: 'vite-plugin:build',
  hook: 'closeBundle'
```

Note that if I remove config.yaml, both npm run dev and npm run prod work as expected

Reply:
Perhaps we should give users the option to run the upgrade and migration separately?

Unless we're planning to drop support for sites-config soon

Reply:
I fixed the prod error. It was caused by checking for null or empty instead of checking truthy.

However the multiple documents for locale error remains